cfme/v2v/migrations.py

```diff
@@ -288,6 +288,31 @@ class MigrationDashboardView(BaseLoggedInPage):
     migration_plans_completed_list = MigrationPlansList("plans-complete-list")
     infra_mapping_list = InfraMappingList("infra-mappings-list-view")
     migr_dropdown = MigrationDropdown(text="Not Started Plans")
+    # TODO: XPATH requested to devel (repo:miq_v2v_ui_plugin issues:415)
+    progress_bar = MigrationProgressBar(locator='.//div[3]/div/div[3]/div[3]/div/div')
```

Comment:
REQUIRED : The locator should be more specific .May be there is an id that you can use ?

Reply:
@sshveta, there is no unique xpath as of now for progress bar div, `id`s requested there as per above TODO.

Reply:
:+1: